### PR TITLE
feat: CIP-95

### PR DESCRIPTION
**Pull request by vladimirvolek**
This is a basic implementation of CIP-95 - https://github.com/cardano-foundation/CIPs/pull/509.
Initial implementation by @Ryun1. 🤠 🪄

PR is still in progress.

Some additional changes to make the project work correctly.

- Fixed lint task (ignored build, node_modules etc.)
- Fixed dependencies
- Updated some old packages
- Fixed imports
- Removed unused tsconfig.json

```
window.cardano.nami.enable({
    extensions: ['cip-95'], // case-insensitive
});
```

PS: *Any kind of feedback is welcome* 💖

### CODE REVIEW

**Comment by mmahut** [comment]
In order to have this release-ready for the testnets, we need to update [cardano_multiplatform_lib](https://github.com/spacebudz/lucid/tree/main/src/core/libs/cardano_multiplatform_lib) to support conway.

**Comment by mmahut** [comment]
> In order to have this release-ready for the testnets, we need to update [cardano_multiplatform_lib](https://github.com/spacebudz/lucid/tree/main/src/core/libs/cardano_multiplatform_lib) to support conway.

[Work in progress](https://github.com/spacebudz/lucid/compare/conway).

**Comment by Ryun1** [comment]
There are a couple issues with my original approach as seen in this PR, I have fixed these now in my [demos implementation](https://github.com/Ryun1/cip95-demos-wallet) but this has a lot more changes which aren't useful. 

I intend to rebase this branch with the Nami relevant changes this week 🤓.

File: src/ui/app/components/transaction.jsx [hunk 1]
```diff
@@ -331,6 +331,8 @@ const TxDetail = ({ displayInfo, network }) => {
                       return 'https://testnet.cardanoscan.io/transaction/';
                     case NETWORK_ID.preview:
                       return 'https://preview.cexplorer.io/tx/';
+                    case NETWORK_ID.preview:
```
Comment (Ryun1) [inline src/ui/app/components/transaction.jsx:334]:
```suggestion
                    case NETWORK_ID.sanchonet:
```

Comment (Ryun1) [reply src/ui/app/components/transaction.jsx]: oops this is wrong!

File: src/ui/app/components/transaction.jsx [hunk 2]
```diff
@@ -331,6 +331,8 @@ const TxDetail = ({ displayInfo, network }) => {
                       return 'https://testnet.cardanoscan.io/transaction/';
                     case NETWORK_ID.preview:
                       return 'https://preview.cexplorer.io/tx/';
+                    case NETWORK_ID.sanchonet:
```
Comment (Ryun1) [inline src/ui/app/components/transaction.jsx:334]:
```suggestion
                    case NETWORK_ID.sancho:
```

File: src/ui/store.jsx [hunk 3]
```diff
@@ -175,6 +175,8 @@ const StoreInit = ({ children }) => {
                       return 'Preview';
                     case NETWORK_ID.preprod:
                       return 'Preprod';
+                    case NETWORK_ID.sanchonet:
```
Comment (Ryun1) [inline src/ui/store.jsx:178]:
```suggestion
                    case NETWORK_ID.sancho:
```

File: src/api/extension/index.js [hunk 4]
```diff
@@ -1367,12 +1379,9 @@ export const createAccount = async (name, password, accountIndex = null) => {
         paymentAddr: paymentAddrTestnet,
         rewardAddr: rewardAddrTestnet,
       },
-      [NETWORK_ID.sancho]: {
```
Comment (Ryun1) [inline src/api/extension/index.js]: Broke this too!

File: src/ui/app/components/transaction.jsx (hunk 2 above)
Comment (Ryun1) [reply src/ui/app/components/transaction.jsx:334]: Fixed in PR

File: src/api/extension/index.js [hunk 5]
```diff
@@ -1484,11 +1511,23 @@ export const getNativeAccounts = (accounts) => {
   return nativeAccounts;
 };
 
-export const indexToHw = (accountIndex) => ({
```
[the remark this answers is not in the document]
Comment (Ryun1) [reply src/api/extension/index.js:1487]: Fixed in PR

File: src/api/extension/index.js (hunk 4 above)
Comment (Ryun1) [reply src/api/extension/index.js]: Fixed in PR

**Comment by Ryun1** [comment]
@rhyslbw I will be re-doing this PR, via my own fork, so please feel free to close this one :)

**Pull request closed without merging**